docs/pages/connect-your-client/tsh.mdx

```diff
@@ -842,5 +842,59 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+# Interactive Recording
+
+Within the Teleport cluster the `ssh` and `Kubernetes` recorded sessions are available for listing and replay for authorized users via `tsh`.
+The same listings including Desktops are available in the Web Console under the Audit Log in the Activity section.
+
+## List Recordings and Play
+
+Get the list of recordings.  Without specifying a time period the past 24 hour recordings are shown.
+```code
+$ tsh recordings ls
+ID                                   Type Participants Hostname Timestamp           
+------------------------------------ ---- ------------ -------- ------------------- 
+b0a04442-70dc-4be8-9308-7b7901d2d600      jeff         dev       Nov 26 16:36:16 UTC 
+c0a02222-70dc-4be8-9308-7b7901d2d600 kube alice                  Nov 26 20:36:16 UTC 
+d0a04442-70dc-4be8-9308-7b7901d2d600      navin        test      Nov 26 16:36:16 UTC 
+```
+
+If no type is specified that is a `ssh` session.
+
+To Play sessions type `tsh play <sessionid>`.  The screen will reset playing the interactive session with the timestamp in the upper right.
+
+```code
+$ tsh play c0a02222-70dc-4be8-9308-7b7901d2d600
+root@ubuntu:/# df                                                                     2022-11-26T21:20:52.715Z
+Filesystem     1K-blocks    Used Available Use% Mounted on
+overlay         20959212 4387284  16571928  21% /
+tmpfs              65536       0     65536   0% /dev
+tmpfs            1982720       0   1982720   0% /sys/fs/cgroup
+/dev/nvme0n1p1  20959212 4387284  16571928  21% /etc/hosts
+shm                65536       0     65536   0% /dev/shm
+tmpfs            3410436      12   3410424   1% /run/secrets/kubernetes.io/serviceaccount
+tmpfs            1982720       0   1982720   0% /proc/acpi
+tmpfs            1982720       0   1982720   0% /sys/firmware
+root@ubuntu:/# exit
+exit
+
+
+end of session playback
+```
+
+### Filtering Recorded sessions
+
+Multiple options are available to get recordings from specific days.  The `--from-utc` and `--to-utc` parameters
```

Comment:
Don't forget the `--last` flag.

But on second thought - this is a pretty high level page. I would keep this page brief and move a more detailed reference to docs/pages/reference/cli.mdx.

Reply:
added these in the CLI reference